blog/config/secrets.yml

```diff
@@ -0,0 +1,22 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
This file should be a `.secret.yml.example` and the `.secret.yml` should be removed from git and put in the Git `.ignore` file.

Remember it for the next projects.